CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - Maintenance CLI command for node management has been added. Currently the only
   supported command is `clear-cache` which clears node message cache. (#676)
 
+- Network connections are now encrypted using
+  [Noise Protocol](https://noiseprotocol.org/). (#678)
+
```

Comment:
It seems that the feature has the breaking changes. Exonum nodes that compiled with old version can't connect to nodes that compiled with the current.